llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -485,8 +485,8 @@ class MUBUF_Load_Pseudo <string opName,
                          list<dag> pattern=[],
                          // Workaround bug bz30254
                          int addrKindCopy = addrKind,
-                         RegisterClass vdata_rc = getVregSrcForVT<vdata_vt>.ret,
-                         RegisterOperand vdata_op = getLdStVDataRegisterOperand<vdata_rc, isTFE>.ret>
+                         RegisterOperand vdata_rc = getVregSrcForVT<vdata_vt>.ret,
```

Comment:
Nit: This is not a RegisterClass anymore, so the name vdata_rc doesn't make much sense. This field only looks to be used in calculating vdata_op, and can probably be removed.

Reply:
Refined in bb3a515f95513571886bb900d4ddefb9ea5b4bad.